File: breakpad-build/src/main/java/com/sogou/translate/breakpad/InfoHelperImpl.java
package com.sogou.translate.breakpad;

import android.util.Log;

public class InfoHelperImpl implements InfoHelper {

    @Override
    public void onNativeCrash(String keyInfo) {
        Log.d("feifei","-----onNativeCrash:"+Thread.currentThread().getName()+",keyInfo:"+keyInfo);
        ShellUtils.execCommand("logcat -d > /sdcard/logfeifei.log",false);
        Log.d("feifei","-----onNativeCrash finished!");

    }
}

File: crashcatcher/src/main/java/com/sogou/iot/arch/crash/InfoHelperImpl.kt
package com.sogou.iot.arch.crash

import com.sogou.translate.breakpad.InfoHelper


class InfoHelperImpl(var nativeCrashHappen:((crashInfo:String)->Unit)) : InfoHelper {

    override fun onNativeCrash(keyInfo:String) {
        nativeCrashHappen.invoke(keyInfo)

    }

}
